Lib/test/test_importlib/namespace_pkgs/project4/homonym.py

```diff
@@ -0,0 +1 @@
+attr = 'homonym'
```

Comment:
Because I think this is an error do to a bad function call when finding a namespace package, this setup isn't necessary.